$NetBSD: patch-ah,v 1.5 2012/10/16 07:31:08 makoto Exp $

 Following chunk existed from old days, and no idea, sorry (mef@).

--- configure.orig	2012-08-08 17:04:57.000000000 +0900
+++ configure
@@ -5331,7 +5331,7 @@
 fi
 
 if test x$with_libpaper != xno; then
-    LIBS="$LIBS -lpaper"
+    LIBS="$LIBS -L${LPLIBDIR} -Wl,-R${LPLIBDIR} -lpaper"
 
 $as_echo "#define USE_LIBPAPER 1" >>confdefs.h
 
